src/v/transform/logging/tests/log_manager_test.cc

```diff
@@ -196,6 +196,10 @@ TEST_F(TransformLogManagerTest, LargeBuffer) {
         ss::maybe_yield().get();
     }
 
+    // Drain the task queue so that we don't get races where this advance should
+    // trigger the flush, but there is currently a pending flush, which doesn't
+    // acuse the advance_clock to actually trigger a flush.
+    tests::drain_task_queue().get();
```

Comment:
Should this just go in the `advance_clock` helper?

Reply:
Unclear, there might be a situation where you don't? I did a quick check over the other usages and I don't think there are more cases where this could happen. I think I'll leave it, for lack of a reason to not 🤷 If you have strong opinions I'm happy to make the change.

Reply:
>strong opinions

Nah, I think you're right.